components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+* Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* you may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.idp.mgt.ui.util;
+
+/**
+ * This class is used to keep the identity provider management UI related constants.
+ */
+public class IdPManagementUIConstants {
+
+    private IdPManagementUIConstants(){
+    }
+
+    public static final String SCIM_PROTOCOL_VERSION = "scim-version";
+    public static final String SCIM_VERSION = "scim1";
```

Comment:
shall we change the constant name to SCIM_VERSION_1 or similar name since this is specific to version 1?